apps/services/endorsements/api/src/app/modules/endorsementList/endorsementList.controller.ts

```diff
@@ -30,6 +36,10 @@ export class EndorsementListController {
     private readonly endorsementListService: EndorsementListService,
   ) {}
 
+  @ApiOkResponse({
+    description: 'Finds all endorsement lists belonging to a given tag',
+    type: [EndorsementList],
+  })
```

Comment:
Nice to add descriptions

Reply:
List of List? :)

Reply:
Maybe we could have found a better name for this 🤔  but the endorsement list contains many endorsements.
This endpoint returns a lists ef endorsement lists